src/acts.js

```diff
@@ -772,4 +772,45 @@ export default {
             .then(pluckData)
             .then(parseDateFields);
     },
+
+    /**
+     * Gets list of act webhooks.
```

Comment:
In new texts we should write "actors". In fact, we should update this in all texts.